src/components/Cards/PriceFeed/index.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { Card, Col, Input, Row, Tooltip } from "antd";
+import { InfoCircleOutlined } from "@ant-design/icons";
+
+const SOL_VALUE = "$ 50.00";
+
+export const PriceFeedCard = () => {
+  return (
+    <Card title="Price" className="cards">
+      <Row>
+        <Col span={2} className="align-to-input">
+          SOL
+        </Col>
+        <Col span={8}>
+          <Input
+            suffix={
+              <Tooltip title={"Current SOL value to USD is " + SOL_VALUE + "."}>
```

Comment:
Please connect to the completed SOL price oracle in $ZeroInterestDeFi/anchor/oracles/pyth.